types/sharp-timer/index.d.ts

```diff
@@ -1,52 +1,323 @@
-// Type definitions for sharp-timer 0.3
+// Type definitions for sharp-timer library. 1.0
 // Project: https://github.com/afractal/SharpTimer
 // Definitions by: Hermes Gjini - afractal <https://github.com/afractal/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export type ElapsedEvent = () => void;
-export type ElapsingEvent = (intervalValue: number) => void;
-
-export class Timer {
-    private _enabled;
-    private _stopped;
-    private _interval;
-    private _intervalElapsedEvents;
-    private _intervalElapsingEvents;
-    constructor(interval: number);
-    readonly enabled: boolean;
-    readonly stopped: boolean;
-    interval: number;
-    start(): void;
-    pause(): void;
-    resume(): void;
-    stop(): void;
-    onIntervalElapsed(intervalElapsedHandler: ElapsedEvent): void;
-    onIntervalElapsing(intervalElapsingHandler: ElapsingEvent): void;
-    toString(): string;
-    private getDoubleDigit(number);
-    private checkForValidInterval(interval);
+/// <reference types="node" />
+
+declare global {
+    namespace SharpTimer {
+        const millisPerSecond: 1000;
```

Comment:
It's tempting to provide these types globally as a convenience since they exist at compile-time only, but it contravenes the module system design in the same way that global values do. These constants should just be exported normally and then explicitly imported by the tests.